common/logger/log.yml

```diff
@@ -0,0 +1,36 @@
+# This file is yaml style config, can make testing the logger more easily.
```

Comment:
Where is it used?

Reply:
It is the yaml style of the rust code in `init` function in `lib.rs`, not used here, but makes it easier to understand the code and test new logger config.

If someone wants to have more customized logger config other than that in the `config.toml` file, he/she may need this.